### ISSUE TITLE
Adding SVG/GIF images in the post will break when zoom in

### ISSUE THREAD

**Issue by tiddler** [narative/gatsby-theme-novela#67]
**Version of Novela you are using**
0.3.6

**Describe the bug**
When the post contains SVG or GIF images, it is rendered correctly but will break the page when clicking on any image to zoom in.

**To Reproduce**

1. Add any SVG/GIF images in the post, for example

   ```html
   <!-- any svg image -->
   <div>
     <img src="./images/foo.svg" />
   </div>
   <!-- or -->
   ![svg](./images/foo.svg)
   <!-- or any gif image-->
   <div>
     <img src="./images/bar.gif" />
   </div>
   <!-- or -->
   ![bar](./images/bar.gif)
   ```

2. Open the page and zoom in any image.

**Expected behavior**
The page should not break.

**Screenshots**
![image](https://user-images.githubusercontent.com/9299201/63107117-5a04d380-bf52-11e9-9a5c-7c0f08af3cab.png)

**Analysis**

1. in `gatsby-remark-image` [source code](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-remark-images/src/index.js#L360), it is actually intented because `SVG` is already responsive, and they cannot  handle `GIF`.
2. When creating the page, these images are not processed but just copied.
3. In the Chrome Dev Tools, we can see these images are rendered differently.
  
![image](https://user-images.githubusercontent.com/9299201/63112705-e1584400-bf5e-11e9-9b0a-56c437bad703.png)

4. Then  `img.previousElementSibling`  is `null` for GIF and SVG and raises the error.

Although it looks like the issue should be handled by the `gatsby-remark-image` plugin, but the code here which assumes sibling is always not null can be a bad practice. Personally I suggest this one-line fix in `src/components/Image.Zoom.tsx` as a quick solution.

```typescript
images.map(img => {
  if (img.previousElementSibling && img.previousElementSibling.tagName === "DIV") {
    img.previousElementsibling.style.background = background;
  }
})
```

I am curious about your idea.

**Comment by brotzky** [comment]
Agreed, this is a fairly easy one-ish line change.
Thanks for reporting! Forgot to test different file formats.

**Comment by brotzky** [comment]
@tiddler fixed in v0.3.7
